capabilities_matrix/_topics/infrastructure_providers.md

```diff
@@ -2,61 +2,61 @@
-| VM Genealogy                                                 | ✅      | ✅          | ✅    | ✅                   |
-| Capacity & Utilization                                       | ✅      | ✅          | ❌    | ✅                   |
-| VM Event Timelines                                           | ✅      | ✅          | ✅    | ❌                   |
-| Infrastructure Event Timelines                               | ❌      | ❌          | ❌    | ✅                   |
-| Reporting                                                    | ✅      | ✅          | ✅    | ✅                   |
-| Right Sizing                                                 | ✅      | ✅          | ❌    | ❌                   |
-| Chargeback by Allocation                                     | ✅      | ✅          | ✅    | ❌                   |
-| Chargeback by Usage                                          | ✅      | ✅          | ❌    | ❌                   |
-| Automation Work Flows                                        | ✅      | ✅          | ✅    | ❌                   |
-| Tag Mapping from Provider                                    | ❌      | ❌          | ❌    | ❌                   |
-| Tag Mapping to Provider                                      | ❌      | ❌          | ❌    | ❌                   |
-| VM Compliance Enforcement                                    | ✅      | ✅          | ✅    | ❌                   |
-| VM Policy Enforcement                                        | ✅      | ✅          | ✅    | ❌                   |
-| Alerts - Real Time                                           | ✅      | ❌          | ❌    | ❌                   |
-| Alerts - VM Value Changed                                    | ✅      | ✅          | ❌    | ❌                   |
-| Alerts - VM Reconfigured                                     | ✅      | ✅          | ❌    | ❌                   |
-| Integrate with Service Catalogs                              | ✅      | ✅          | ✅    | ❌                   |
+| General Features                                             | vSphere | oVirt / RHV | SCVMM | OpenStack undercloud | IBM Power HMC |
+| ------------------------------------------------------------ | ------- | ----------- | ------| -------------------- | ------------- |
+| Relationship Discovery                                       | ✅      | ✅          | ✅    | ✅                   | ✅            |
+| Drift Comparison                                             | ✅      | ✅          | ✅    | ✅ (Nodes)           | ❌            |
+| VM Genealogy                                                 | ✅      | ✅          | ✅    | ✅                   | ❌            |
+| Capacity & Utilization                                       | ✅      | ✅          | ❌    | ✅                   | ❌            |
+| VM Event Timelines                                           | ✅      | ✅          | ✅    | ❌                   | ❌            |
```

Comment:
Since you collect events in your provider you will have VM Event Timelines, that is handled by core automatically for you